DirectEmployers/MyJobs
# [PD 3117] reorder navigation

## What's this PR do?
Reorders the navigation on the side menu inside of NUO. Changes Outreach Records ABOVE Inbox Management

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the NUO application

## How should this be manually tested?
Pull down the repository. Go to NUO and you will see navigation on the right hand side. Inbox Management should now be BELOW Outreach Records whereas before it was above it

## Any background context you want to provide?
